cloudinit/util.py

```diff
@@ -1814,14 +1831,8 @@ def uptime():
                 uptime_str = contents.split()[0]
         else:
             method = 'ctypes'
```

Comment:
Let's add a comment under the else clause to indicate that we're in a FreeBSD code path.